spring/src/main/java/org/axonframework/spring/config/annotation/SpringHandlerDefinitionBean.java

```diff
@@ -39,10 +41,12 @@
  * @see ClasspathHandlerDefinition
  * @since 3.3
  */
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.